addons/docs/src/mdx/mdx-compiler-plugin.js

```diff
@@ -19,12 +19,16 @@ function getAttr(elt, what) {
 }
 
 const isReserved = name => RESERVED.exec(name);
+const startsWithNumber = name => /^\d/.exec(name);
 
 const sanitizeName = name => {
   let key = camelCase(name);
   if (isReserved(key)) {
     key = `${key}Story`;
   }
+  if (startsWithNumber(key)) {
```

Comment:
Probably put this before the `isReserved` check?

Reply:
why?

Reply:
fixed

Reply:
Add "Story" to the end of a name is never going to make it start with a number.

Adding `_` to the start of a name is probably never going to make it a reserved word, but I don't know off the top of my head what all the reserved words are so it seems safer that way round.